Library/Formula/apache-archiva.rb

```diff
@@ -0,0 +1,63 @@
+class ApacheArchiva < Formula
+  desc "The Build Artifact Repository Manager"
+  homepage "https://archiva.apache.org/"
+  url "http://ftp.unicamp.br/pub/apache/archiva/2.2.0/binaries/apache-archiva-2.2.0-bin.tar.gz"
```

Comment:
Please use this canonical Apache URL:
https://archive.apache.org/dist/archiva/2.2.0/binaries/apache-archiva-2.2.0-bin.tar.gz